content/Product/OpenshiftMonitoring/faq.md

```diff
@@ -0,0 +1,143 @@
+# Debugging the default OpenShift 4.x in-cluster Monitoring: Platform Monitoring and User Workload Monitoring Products
+
+## Overview
+
+This serves as a collection of resources that relate to FAQ around configuring/debugging the in-cluster monitoring stack. Particularly it applies to two Openshift Projects:
+
+* [Platform Cluster Monitoring](https://docs.openshift.com/container-platform/latest/monitoring/understanding-the-monitoring-stack.html#understanding-the-monitoring-stack_understanding-the-monitoring-stack) (??)
+* User Workload Monitoring (link)
```

Comment:
we could link to https://docs.openshift.com/container-platform/latest/monitoring/enabling-monitoring-for-user-defined-projects.html